### ISSUE TITLE
Top-level menu bar (File, Time Control, etc.) broken in the native Mac version of HawkEye

### ISSUE THREAD

**Issue by mmbell** [nsf-lrose/lrose-blaze#3]
The top-level menu bar (File, Time Control, etc.) works fine in the Docker version, but is unresponsive in the Mac version. It is visible but doesn't respond to clicks.

**Comment by leavesntwigs** [comment]
This seems to describe the behavior I've noticed (https://github.com/robotology/yarp/issues/457).

**Comment by mmbell** [comment]
Yes, I can reproduce this behavior on my Mac as well running High Sierra 10.13.5 and compiled with Qt 5.11.1 from Homebrew. If you focus on another app or click the Desktop then go back it works.

From that discussion it seems like it may be library issue.

**Comment by leavesntwigs** [comment]
Yes, building HawkEye as a bundle seems to fix the problem of the top-level menus being unresponsive.  I used qmake to create the bundle.  I'll have to do some more testing and try to create a .dmg file for packaging and deployment.

**Comment by leavesntwigs** [comment]
I've created a .dmg file and installed it on a separate Mac.  It installs easily and continues to work well, with responsive top-level menus.  I am just working through some icon issues, then I will publish the installable package.
